src/pages/Swap/UniswapXOptIn.tsx

```diff
@@ -111,40 +111,38 @@ const OptInContents = ({
 
   const containerRef = useRef<HTMLDivElement>()
 
-  const wrapTrace = (children: JSX.Element) => {
+  if (isSmall) {
```

Comment:
I think you want `<Trace>` to enclose everything, and put the conditional within it, so that you don't double-log if a user modifies their screen size.